pdfarranger/pdfarranger.py

```diff
@@ -114,7 +114,7 @@
 from .iconview import IconviewCursor
 from .iconview import IconviewDragSelect
 from .config import Config
-from .core import img2pdf_supported_img
+from .core import img2pdf_supported_img, PageAdder, PDFDocError, PDFRenderer
 from .core import *
```

Comment:
TODO: Check if we can entirely remove the * import

Reply:
I would play it safe and import everything listed in `__all__` in core.py

```suggestion
from .core import img2pdf_supported_img, Page, PageAdder, PDFDoc, PDFDocError, PDFRenderer
```

Reply:
Good idea!

BTW: How do you suggest this change so that I can commit it right away? Have seen that every now and then but it seems to be well hidden every time I review code...

Reply:
Well that caused some other warnings... Maybe we should stuck with the strand accept those?